MANIFEST.in

```diff
@@ -10,3 +10,4 @@ include CHANGELOG.md
 include AUTHORS.txt
 include NOTICE.txt
 include bootstrap.py
+recursive-include spyder *.py

```

Comment:
This is not necessary. Please revert it.

Reply:
Fixed. (some tests failed before without it 🤷 , this has happened in other PRs also)

Reply:
Ok, they are failing again @ccordoba12 

![image](https://user-images.githubusercontent.com/3627835/90181517-30a1a200-dd76-11ea-8c82-501dc0729c95.png)